tests/schemes/test_thrift.py

```diff
@@ -1162,6 +1163,6 @@ def test_headers_should_be_a_map_of_strings(headers):
 
     with pytest.raises(ValueError):
         yield tchannel.thrift(
-            request=True,
+            request=mock.MagicMock(),
```

Comment:
This original one breaks test since the thrift method will access request.result_type.  If you pass a True, it will break.